platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLFCustoms.java

```diff
@@ -73,8 +75,10 @@
             VIEW_TAB_DISPLAYER_UI, "org.netbeans.swing.laf.flatlaf.ui.FlatViewTabDisplayerUI", // NOI18N
             SLIDING_BUTTON_UI, "org.netbeans.swing.laf.flatlaf.ui.FlatSlidingButtonUI", // NOI18N
 
-            EDITOR_TABSCOMPONENT_BORDER, DPISafeBorder.matte(1, 1, 1, 1, editorContentBorderColor),
-            EDITOR_TOOLBAR_BORDER, DPISafeBorder.matte(0, 0, 1, 0, editorContentBorderColor),
+            EDITOR_TABSCOMPONENT_BORDER, BorderFactory.createEmptyBorder(),
+            EDITOR_TOOLBAR_BORDER, new CompoundBorder(
+                DPISafeBorder.matte(0, 0, 1, 0, editorContentBorderColor),
+                BorderFactory.createEmptyBorder(1, 0, 1, 0)),
```

Comment:
nitpick: All other key/values use a single line. I think it would be easier to read to do the same here.

Reply:
Sure, incorporated into the new PR https://github.com/apache/netbeans/pull/4335